src/services/user.service.ts

```diff
@@ -0,0 +1,53 @@
+import { ModelCtor, Op } from 'sequelize';
+import { Inject, Service } from 'typedi';
+
+import { Config } from '../config';
+import { CreateActionUser, UpdateActionUser } from '../interfaces';
+import { IUserInstance } from '../models';
+
+@Service()
+export class UserService {
+    constructor(@Inject(Config.injectionToken.userModel) private userModel: ModelCtor<IUserInstance>) {}
```

Comment:
It makes sense to introduce an abstraction over `ModelCtor<IUserInstance>`, so that `UserService` doesn't depend on a specific implementation of data access layer (possible implementation are limited to ones supported by `sequelize` at the moment). Otherwise we'll have to modify service layer if we want to use other (e.g. in-memory) implementation of DAL